include/internal/catch_compiler_capabilities.h

```diff
@@ -220,6 +228,9 @@
 #    define CATCH_CONFIG_CPP11_TO_STRING
 #endif
 
+#if defined(CATCH_INTERNAL_CONFIG_CPP17_OPTIONAL) && !defined(CATCH_CONFIG_NO_CPP17_OPTIONAL) && !defined(CATCH_CONFIG_CPP17_OPTIONAL)
+#endif
```

Comment:
Where is the `#define CATCH_CONFIG_CPP17_OPTIONAL` gone?

Reply:
My mistake. I guess I was too tired yesterday.